tests/transforms/test_broadcast_expand.py

```diff
@@ -107,22 +112,27 @@ def test_expansion(self, params, size, obs, exp_fn):
         assert qml.math.allclose(result, exp_fn(*params))
 
     def test_without_broadcasting(self):
+        """Test that an error is raised when applying the transform to a non-broadcasted tape."""
         tape = make_tape(0.2, 0.1, 0.5, [qml.PauliZ(0)])
         with pytest.raises(ValueError, match="The provided tape is not broadcasted."):
             qml.transforms.broadcast_expand(tape)
 
     @pytest.mark.autograd
     @pytest.mark.filterwarnings("ignore:Output seems independent of input")
-    @pytest.mark.parametrize("params, size", parameters_and_size)
+    @pytest.mark.parametrize("params", parameters)
     @pytest.mark.parametrize("obs, exp_fn", observables_and_exp_fns)
-    def test_autograd(self, params, size, obs, exp_fn):
+    @pytest.mark.parametrize("gradient_fn", [qml.gradients.param_shift, "backprop"])
```

Comment:
This parametrization is new @albi3ro. I added it for Autograd, Torch and JAX (but not for TF for cost reasons: The TF tests in this class take the majority of the time for all tests in this file).
Previously, `gradient_fn` was fixed to be `param_shift`. With `"backprop"` and for JAX with `use_jit=True`, the test fails on master.